ci/docker/runtime_functions.sh

```diff
@@ -1296,6 +1296,16 @@ nightly_scala_demo_test_cpu() {
     bash bin/run_im.sh
 }
 
+nightly_estimator() {
+    set -ex
+    cd /work/mxnet/tests/nightly/estimator
+    export PYTHONPATH=/work/mxnet/python/
+    python test_estimator_cnn.py --type gpu
+    python test_sentiment_rnn.py --type gpu
+    python test_estimator_cnn.py --type cpu
+    python test_sentiment_rnn.py --type cpu
```

Comment:
we are running python scripts instead of nosetests, but both will throw error 
I'm combining these 4 tests, although running them separately locally take less than 5 min, but creating a separate job and run on CI takes long time. So I'm adding total 1 additional job to run.